### PR TITLE
extra debug

### PR DESCRIPTION
**Pull request by benjamin-confino**
Adds some extra debug that would have been useful for a recent PMR

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/BeanDeploymentArchiveImpl.java [hunk 1]
```diff
@@ -303,6 +311,9 @@ private boolean isAccessibleBean(Class<?> beanClass) {
     }
 
     private Set<String> scanForBeanClassNames() throws CDIException {
+        if ( TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled() ) {
```
Comment (tevans78) [inline dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/BeanDeploymentArchiveImpl.java]: I think you want to either use both entry and exit OR just use debug

**Review by tevans78 — APPROVED** [review]
Just need to update the copyright date

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#run-libby-bot

**Pull request merged**